src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVA.jsx

```diff
@@ -30,22 +31,25 @@ export default function DetailsVA({ appointment, facilityData }) {
   const isCovid = appointment.vaos.isCOVIDVaccine;
   const header = formatHeader(appointment);
   const isPhone = appointment.vaos.isPhoneAppointment;
+  const serviceType = appointment.vaos.apiData.serviceType;
+  const typeOfCare = getTypeOfCare(serviceType)?.name;
 
   return (
     <>
       <Breadcrumbs>
         <Link to={`/va/${appointment.id}`}>Appointment detail</Link>
       </Breadcrumbs>
-
       <h1>
         <AppointmentDateTime appointment={appointment} />
       </h1>
-
       <StatusAlert appointment={appointment} facility={facility} />
-
+      {typeOfCare && (
+        <div className="vads-u-padding-top--2p5">
+          <b>Type of care:</b> {typeOfCare}
+        </div>
+      )}
```

Comment:
Type of care should be an h2 element so screen readers can tag it has a heading allowing users to audibly scan through headings. It's part of 508. There is also a [VA design system guide](https://design.va.gov/utilities/) that will be helpful with styling.
```suggestion
       {typeOfCare && (
        <>
          <h2 className="vads-u-font-size--base vads-u-font-family--sans vads-u-margin-bottom--0 vads-u-display--inline">
            Type of care:
          </h2>
          <div className="vads-u-display--inline"> {typeOfCare}</div>
        </>
      )}
```

Reply:
@jenniemc - Thank you!.....I am so new to this being such a hard requirement, can we make this one of our COP topics to talk about?

Reply:
Absolute!